integrations/google/creds.go

```diff
@@ -123,6 +123,11 @@ func (h handler) finalize(ctx context.Context, c sdkintegrations.ConnectionInit,
 		return
 	}
 
```

Comment:
@daabr
- is there any other integration that could potentially modify the list of vars via Set?
- could we check this in c.Finalize and error to prevent/report errors?

Reply:
Discussed offline